### ISSUE TITLE
[Bug Report] Cards don't fill empty vertical space in Grid

### ISSUE THREAD

**Issue by uodami** [vuetifyjs/vuetify#11810]
### Environment
**Vuetify Version:** 2.3.3
**Last working version:** 1.5.24
**Vue Version:** 2.6.11
**Browsers:** Chrome 75.0.3770.100
**OS:** Windows 10

### Steps to reproduce
Make a grid containing two columns, the first one containing another column-direction grid and the second one consisting of only a simple column.
Set align to stretch on the row and on the card.

### Expected Behavior
The column stretches to fill the vertical space
The card stretches to fill the available vertical space (equal to the columns' sizes).

### Actual Behavior
The card does not stretch to fill the available space.

### Reproduction Link
<a href="https://codepen.io/uodami/pen/RwrMZMo" target="_blank">https://codepen.io/uodami/pen/RwrMZMo</a>

### Other comments
Here is a codepen that works using the old grid, in version 1.5
https://codepen.io/uodami/pen/yLeKvBr

**Comment by 185driver** [comment]
I was looking at this issue to see if I could understand it better. As best I can tell:

1. The submitter's v2 codepen repro is materially different than the v1.5 repro so I didn't find the v2 repro useful. Instead, I recreated the v1.5 repro using v2.3.3 [https://codepen.io/185driver/pen/ExPLKwP](https://codepen.io/185driver/pen/ExPLKwP). v2 no longer uses the `grid-list-{size}` prop for `v-row` so I fudged slightly on that to make my repro match but it's an inconsequential change. I also left out the `flex-wrap` props from the v1.5 repro as they don't seem to do anything related to the issue.

1. I did not find grid behavior to be different in v2 vs v1.5, but v-card #3 does not stretch across the main-axis in the v2 repro like it does in v1.5. Adding `class="flex-grow-1"` to v-card #3 seemed to care for the issue. That's the only material change I made.

1. The submitter mentions that the card does not stretch to fill the available vertical space (its column). Using my own recreated v2 repro, I found that the card (#3) **does** stretch to fill the available vertical space, but instead it does not fill the available **horizontal space**.

Whether there's really an issue here to fix, I can't 100% say, but to me the issue to be resolved (if any) does not seem to be the one the submitter explains above.
